README.md

```diff
@@ -1,70 +1,243 @@
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
 
-### Advanced Configuration
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
 
-### Deployment
+## 💻 Getting Started <a name="getting-started"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+- A new developer can get the project from my Github repo link i shall provide.
 
-### `npm run build` fails to minify
+- To get a local copy up and running, follow these steps.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+- go to https://github.com/stephenkati/Unsplash-Image-Hub.git
+
+- clone or download the project
+
+- Run `npm install`
+- Run `npm start`
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A code editor, preferably VSCode. Any other code editor is okay.
+
+- Browser
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Run `git clone https://github.com/stephenkati/Unsplash-Image-Hub.git`
+- cd `space-traveler-s-hub`
+
+### Install
+
+Install this project with:
+
+- Download the project from github and run it in your browser.
+
+- Install dependencies.
+- `npm install`
+
+
+
+### Usage
+
+- To run the project, navigate to the project directory and use a live Server extension in your VSCode to run the project.
+
+Run `npm start` 
+
+
+### Run tests
+
+You can run tests to check for linter errors by running the following commands:
+
+- `npx stylelint "**/*.{css,scss}"` for testing css errors
+- `npx eslint "**/*.{js,jsx}"` for Javascript errors
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Deployment
+
+The project is deployed on [Render](https://image-hub.onrender.com/)
+
+## 👥 Author <a name="authors"></a>
+
+
+👤 **Stephen Katuli**
+
+- GitHub: [@githubhandle](https://github.com/stephenkati)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/stephen-katuli/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[More backround images]**
+- [ ] **[Animations and transitions]**
+- [ ] **[More Javascript Functionality]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+- I am very grateful to [Nelson Sakwa on Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) for the design I used.
+
```

Comment:
- [ ] Please add this line in the `Acknowledgments` section because this is `Acknowledgment` not a `Contribution` so kindly fix it :slightly_smiling_face: